### PR TITLE
Empty null check

### PR DESCRIPTION
**Pull request by spuniani**

### CODE REVIEW

File: src/main/java/com/sarality/form/FormData.java [hunk 1]
```diff
@@ -172,6 +170,13 @@ private String getValue(String fieldName) {
     return fieldValueMap.get(fieldName);
   }
 
+  private String convertEmptyToNull(String value) {
+    if (value.equals("")) {
```
Comment (abhideep) [inline src/main/java/com/sarality/form/FormData.java]: This will throw an NPE when value is null.

Use TextUtils.isEmpty()

Comment (spuniani) [inline src/main/java/com/sarality/form/FormData.java]: in that case, why dont we just replace the null check in the functions with an isEmpty check? Do we need to even convert empty to null?

Comment (abhideep) [inline src/main/java/com/sarality/form/FormData.java]: That works check for TextUtils.isEmpty rather than null checks

Comment (spuniani) [inline src/main/java/com/sarality/form/FormData.java]: done and pushed.

**Pull request merged**